kjourney/src/app/register-user/register-user.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { RegisterUserComponent } from './register-user.component';
+
```

Comment:
need to see some tests

Reply:
will write some unit tests now

Reply:
unable to implement in the timeframe however manual testing passing